index.html

```diff
@@ -444,4 +512,32 @@ <h5 class="name">Sanmugathasan Prasanthan</h5>
         )
     })
 </script>
+<script>
+    (function () {
+        const timeline = document.querySelector(".timeline ol"),
+            elH = document.querySelectorAll(".timeline li > div");
+
+        window.addEventListener("load", init);
+
+        function init() {
+            setEqualHeights(elH);
+        }
```

Comment:
Can be written as:
```suggestion
       $(function () {
           const elH = document.querySelectorAll(".timeline li > div");
           setEqualHeights(elH);
       }
```
btw, make variables meaningful. 
Also, you could move this into a separate js file dedicated to index.html. call the function from the line `506`. We don't need the `IIFE` at line `516` if we move this to an external script.

Reply:
Roger that

Reply:
👍🏻